docs/tutorial/docker.md

```diff
@@ -23,44 +23,72 @@ flowchart LR
     C1(Container)
     C2(Container)
     C3(Container)
+    J{Jupyter-Lab}
+    U((User))
 
     DF -->|build| CI --> C1
     CI --> C2
     CI --> C3
+    C1 --> J
+    U --> J
+    J --> U
 ```
 - Basic Terminology
   - Dockerfile: Instruction set to create image.
   - Image: _Template_ by which a container will be initialized at runtime.  To create image, use the command `docker build`.
   - Container: Isolated computing environment. Each container can be created from image by the command `docker run`.
 
-### Build
+### Build Image
+
+The command below builds `image` tagged `miv_env:01`, based on the instruction set provided in [Dockerfile][url-mivsim-dockerfile]. To see the available images, run `docker images`.
 
 ```bash
 cd MiV-Simulator                         # Change directory to repository
 docker build . --tag miv_env:0.1         # Build image
 ```
 
-This command builds `image` tagged `miv_env:01`, based on the instruction set provided in [Dockerfile](https://github.com/GazzolaLab/MiV-Simulator/blob/main/Dockerfile). To see the available images, run `docker images`.
+### Create Container From Image
 
-### Create Container
+The command below both _initialize_ a container `miv_env:0.1` and _launch_ that container. By default, the container will run the `jupyter-lab` in the background on the port `8888`. User can alther the destination port by setting `<destination port>:8888`.
 
 ```bash
-docker run -p 8888:8888 -it miv_env:0.1  # Launch jupyter-lab
+docker run -p 8888:8888 -it miv_env:0.1
 ```
 
-The above command both _initialize_ a container `miv_env:0.1` and _launch_ that container. By default, the container will run the `jupyter-lab` in the background on the port `8888`. User can alther the destination port by setting `<destination port>:8888`.
-
 To see the running containers, run `docker ps` or `docker container ls`. Too show all containers including the one stopped, pass `-a` or `-all`.
```

Comment:
Just an observation: this feels like scope creep since the documentation shouldn't be the place to teach generic docker skills. My suggestion would be to leave the tailored version of the commands as they apply to MiV and send people to the official documentation for anything more generic.

Reply:
I agree that I should be careful, but I would like to include enough information to run the docker in our MiV-Simulator context without digging into the docker much. I'll remove technical descriptions.